frontend/src/app/commonComponents/Modal.tsx

```diff
@@ -27,8 +28,13 @@ const variantIcons: Record<ModalVariant, IconDefinition> = {
   warning: faExclamationCircle,
 };
 
-const Header: React.FC<{}> = ({ children }) => (
-  <h3 className="modal__heading">{children}</h3>
+const Header: React.FC<{ styleClassNames?: string }> = ({
```

Comment:
Neat! Glad that changing this will fix the other modal DuplicatePatientModal too!